FSJ_django20_project/FSJ/templates/FSJ/coord_awards_list.html

```diff
@@ -0,0 +1,59 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %} {# NOTE YOU MUST LOAD i18n to every HTML page even though it's also loaded to base.html #}
+
+{% block heading %}
+    {# Translators: Awards page home label #}
+    {% trans "Awards" %}
+{% endblock %}
+
+{% block content %}
+<body>
+<div>
+<form action="/coord_awardslist/delete/" method="POST">
+{% csrf_token %}
+<table align = "center">
+ <tr>
+  {# Translators: Action header in awards table #}
+  <th>{% trans "Action" %}</th>
+  {# Translators: Award name header in awards table #}
+  <th>{% trans "Award" %}</th>
+  {# Translators: Award description header in awards table #}
+  <th>{% trans "Description" %}</th>
+  {# Translators: Award value header in awards table #}
+  <th>{% trans "Value" %}</th>
+  {# Translators: Award programs header in awards table #}
+  <th>{% trans "Programs" %}</th>
+  {# Translators: Award years of study header in awards table #}
+  <th>{% trans "Years of Study" %}</th>
+  {# Translators: Award deadline header in awards table #}
+  <th>{% trans "Deadline" %}</th>
+  {# Translators: Award documents needed header in awards table #}
+  <th>{% trans "Documents Required?" %}</th>
+  {# Translators: Award is active header in awards table #}
+  <th>{% trans "Is Active?" %}</th>
+ </tr>
+{% for award in awards_list %}
+    <tr>
+     <td><input type="checkbox"  name="todelete" value="{{award.awardid}}" > </td>
+     <td>{{award.award_name }}</td>
+     <td>{{award.description}}</td>
+     <td>{{award.value}}</td>
+     <td>{{award.programs}}</td>
+     <td>{{award.years_of_study}}</td>
```

Comment:
Can we translate this to the text instead of the numerical value of the YEAR_CHOICES tuples? Since we're translating those integers into text it would be significantly easier to read.

Reply:
Done